CMakeLists.txt

```diff
@@ -275,11 +275,14 @@ IF(WIN32)
 ENDIF(WIN32)
 
 ADD_LIBRARY(kml ${libkml_SOURCES} )
+LIST(APPEND KML_TARGETS kml)
 
 IF(NOT LIBKML_USE_EXTERNAL_KML)
   TARGET_LINK_LIBRARIES(kml expat)
+  LIST(APPEND KML_TARGETS expat)
 ELSE (NOT LIBKML_USE_EXTERNAL_KML)
   TARGET_LINK_LIBRARIES(kml ${EXPAT_LIBRARIES})
+  LIST(APPEND KML_TARGETS ${EXPAT_LIBRARIES})
```

Comment:
Something's not right here: when I try to configure this, I get:

    CMake Error at CMakeLists.txt:384 (export):
      export given target "expat" which is not built by this project.

I'm pretty sure expat should not be an exported target unless the embedded version is being built; when using an external expat, it won't be a target.